src/minio/managers/policy_manager.py

```diff
@@ -410,6 +519,23 @@ def _create_object_operations_statement(
             principal=None,
         )
 
+    def _get_actions_for_permission_level(
+        self, level: PolicyPermissionLevel
+    ) -> List[PolicyAction]:
+        """Get actions for a permission level."""
+        permission_mapping = {
+            PolicyPermissionLevel.READ: [
+                PolicyAction.GET_OBJECT,
+            ],
+            PolicyPermissionLevel.WRITE: [
+                PolicyAction.GET_OBJECT,
+                PolicyAction.PUT_OBJECT,
+                PolicyAction.DELETE_OBJECT,
+            ],
+            PolicyPermissionLevel.ADMIN: [PolicyAction.ALL_ACTIONS],
+        }
```

Comment:
This can be a module or class constant, then I don't think this method is needed

Reply:
👍

Reply:
👍

Reply:
should it throw an exception rather than return read perms if the PolicyPermissionLevel is unknown?

Reply:
:+1: